apps/coscrad-frontend/src/components/COSCRADLogo/COSCRADLogo.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { renderWithProviders } from '../../utils/test-utils';
+import { COSCRADLogo } from './COSCRADLogo';
+
+describe('COSCRADLogo', () => {
+    it('should render the logo image successfully', () => {
+        const { baseElement } = renderWithProviders(<COSCRADLogo />);
+        expect(baseElement).toBeTruthy();
+    });
+    test('the logo image should display', () => {
```

Comment:
Don't use `test`. Stick to `it`- just for consistency. You can also reword the description once you use `it`.

Reply:
done